test/functional/mining_basic.py

```diff
@@ -213,6 +213,12 @@ def assert_submitblock(block, result_str_1, result_str_2=None):
         assert_equal(mining_info['bits'], nbits_str(REGTEST_N_BITS))
         assert_equal(mining_info['target'], target_str(REGTEST_TARGET))
         assert_equal(mining_info['difficulty'], Decimal('4.656542373906925E-10'))
+        assert_equal(mining_info['next'], {
+            'height': 201,
+            'target': target_str(REGTEST_TARGET),
+            'bits': nbits_str(REGTEST_N_BITS),
+            'difficulty': Decimal('4.656542373906925E-10')
```

Comment:
nit:
Could be better to define this value as a const just like you did for nBits and target.

Reply:
Will do if I need to retouch.